src/viam/ur/module/ur_arm_state.cpp

```diff
@@ -414,10 +414,23 @@ std::string URArm::state_::describe_() const {
     return describe_state_(current_state_);
 }
 
+std::string URArm::state_::describe() const {
+    const std::lock_guard lock{mutex_};
+    return describe_();
+}
+
 std::string URArm::state_::describe_state_(const state_variant_& state) {
     return std::visit([](auto& state) { return state.describe(); }, state);
 }
 
+bool URArm::state_::is_current_state_controlled(std::string* description /*= nullptr*/) const {
+    const std::lock_guard lock{mutex_};
+    if (description != nullptr) {
+        *description = describe_();
+    }
+    return current_state_.index() == 1;
```

Comment:
I couldn't figure out a way to put `state_controlled_` here, was there a quick way using `state_variant` that I'm missing or should I include something else? Or is this sufficient for now?